Server/tests/test_keep_running_mode.py

```
@@ -0,0 +1,62 @@
+        # Test that RegisterMessage can be created with keep_server_running=False
+        msg_false = {
+            "project_name": "TestProject",
+            "project_hash": "hash456",
+            "unity_version": "2022.3.0f0",
+            "project_path": "/Test/Path",
+            "keep_server_running": False
+        }
+
+        # Verify the field name and values are correct
+        assert "keep_server_running" in msg_true
+        assert msg_true["keep_server_running"] == True
```

Ruff (0.15.0)

[error] Avoid equality comparisons to `True`; use `msg_true["keep_server_running"]:` for truth checks

Replace with `msg_true["keep_server_running"]`

(E712)